### DISCUSSION TITLE
Allow editing Country and Sector if data not available

### DISCUSSION THREAD

**Discussion by SvbZ3r0** [ghostfolio/ghostfolio#2619] [Ideas]
Release 2.15.0 allows editing the name, asset class and asset sub class of asset profiles with MANUAL data source. This should further allow editing country and sector.
If a ticker does not have this data available through available data sources, we should be able to manually input it.

**Comment by dtslvr** [comment]
Hi @SvbZ3r0

Thank you for your suggestion. Would you like to work on that?

**Reply by SvbZ3r0** [reply]
Hi, I'm not familiar with the tech stack, but I can try! Will look into it this weekend.
Do you have any contributing guidelines?

**Reply by dtslvr** [reply]
Thank you for your interest in contributing.

> Do you have any contributing guidelines?

Please follow common sense and stick to the existing code style.

**Reply by SvbZ3r0** [reply]
Hi!
I spent a couple days looking at the code, but with my lack of experience with the entire tech stack and the lack of documentation, i am in way over my head.
It would be best if someone who's more familiar with the project could pick this up when they can.

**Comment by ziritione85** [comment]
So, how you manually add countries and sectors?

**Reply by dtslvr** [reply]
You could manually create a database entry in the `SymbolProfileOverrides` table and set `countries` in the following format: `[{"code": "IT", "weight": 1}]`.

Same for `sectors`, e.g. `[{"name":"Real Estate", "weight":1}]`

**Reply by ziritione85** [reply]
You can’t doit via admin conotrol/Market Data and edit a manual asset? 

Thanks

**Reply by tvirus97** [reply]
@dtslvr  It would be great `if` this option was available in the web interface because all mutual funds show no country.
